.github/workflows/cdelivery-s3-apps-caller.yml

```
@@ -11,7 +11,7 @@ jobs:
   graasp-deploy-s3-apps-workflow:
     # Replace with repository name
     # todo: replace me
-    name: GRAASP_TEMPLATE_FULL_NAME
+    name: graasp_app_cv_tools
```

```suggestion
    name: Graasp CV App
```
Same for the other workflows